test/snapshots/mac/masTest.js.snap

```diff
@@ -10,13 +10,25 @@ exports[`dev 2`] = `
 {
   "CFBundleDisplayName": "Test App ßW",
   "CFBundleExecutable": "Test App ßW",
-  "CFBundleIconFile": "Test App ßW.icns",
+  "CFBundleIconFile": "icon.icns",
```

Comment:
Will there be any other issues since the name has been changed?

Reply:
No production code was changed in this PR. I was updating a snapshot localling as `masTest` was actually never running in the CI.
https://github.com/electron-userland/electron-builder/pull/8907/files#diff-e5263da2cd1e4118e9e993957ad0609307ef3c828d1a151d28fb9041545a1294L10-L14